module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -101,6 +101,47 @@ namespace sys
         LOG_DEBUG("%s", (GetName() + ":destructor").c_str());
     }
 
+    void logPowerOffReason(SystemManagerCommon::State state)
```

Comment:
Why is it left out of any namespace qualification?

Reply:
fixed